t/page/countries.rb

```diff
@@ -0,0 +1,26 @@
+require 'minitest/autorun'
+require_relative '../../lib/page/countries'
+require 'pry'
+
+CJSON = 'https://cdn.rawgit.com/everypolitician/everypolitician-data/%s/countries.json'.freeze
+SHA   = 'd8a4682f'.freeze
+
+describe 'Countries' do
+  subject do
+    Page::Countries.new
+  end
+
+  it 'has a list of all countries in Everypolitician' do
+    assert_equal subject.countries.first[:name], 'Abkhazia'
+  end
+
+  it 'has a url pointing to the last version of the db' do
+    datasource = CJSON % SHA
+    Everypolitician.countries_json = datasource
```

Comment:
I'm not sure that setting this at this point has the effect you think it's having. And, worse, I'm not sure how to _be_ sure that it does. The logic here might be simpler to follow if *all* the tests were using the same datasource, and this one was just testing that it was the expected one.